.github/workflows/flutter_test.yml

```
@@ -53,79 +54,110 @@ jobs:
               mkdir -p "$(dirname "$test_file")"
               cat <<EOF > "$test_file"
-import 'dart:io';
-import 'package:flutter_test/flutter_test.dart';
-import 'package:flutter/material.dart';
-import 'package:team_kiosk/main.dart';
-import 'package:team_kiosk/${widget_file/lib\//}';
-import 'package:http/http.dart' as http;
-import 'package:flutter/services.dart';
-
-void main() {
+  import 'dart:io';
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 57: could not find expected ':'

(syntax-check)